src/Integrations/Integrations/Lumen/LumenIntegration.php

```diff
@@ -39,6 +39,7 @@ public function init()
         }
 
         $rootSpan = \DDTrace\root_span();
+        $rootSpan->meta[Tag::COMPONENT] = LumenIntegration::NAME;
```

Comment:
Put it after the null check to avoid nullrefs.

Reply:
done.